js/constants/constants.js

```diff
@@ -88,6 +88,21 @@ module.exports = {
     SHADOW: "Shadow"
   },
 
+  TOOLTIPS: {
```

Comment:
yes they'll be localized, didn't know it was in the original scope but I'll be pushing the changes early Thursday.